src/ui/screens/DemocracyScreen.tsx

```diff
@@ -65,8 +65,14 @@ export function DemocracyScreen() {
                 </>
               );
             }}
+            renderSectionFooter={({section}) =>
+              section.data.length < 1 ? (
+                <Card style={globalStyles.paddedContainer}>
+                  <EmptyState subheading="There are no items to display" caption="Please check back for updates" />
```

Comment:
"There are no items to display" is to generic.. It should be "**There are no active referendums**" and "**There are not active proposals**" accordingly